src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -120,7 +120,9 @@
      * Indicate whether the user's hard drive is running out of space for game downloads.
      */
     private final Property<Optional<Warning>> warning;
-
+    private Property<Tooltip> playButtonTooltip;
+    private Property<Package> selectedPackageProperty;
```

Comment:
Are there existing Properties for these things that we can use without making new ones?